bluemira/base/parameter.py

```diff
@@ -326,6 +344,18 @@ def source_history(self):
         """
         return self._source_history
 
+    @property
+    def unit(self):
+        """
+        The unit of the parameter
+        """
+        return self._unit
+
+    def _unit_setup(self, unit: Union[Unit, str, None]):
+        if isinstance(unit, None):
+            bluemira_warn(f"{self.var} has no unit")
```

Comment:
This is going to happen a lot... Could we define a NO_UNIT unit or something? We have so many fractions, ratios, strings, and the like, and I think it will be quite easy to lose consistency between ["n/a", "N/A" and whatever]. Perhaps you've already got something planned!

Reply:
There is a builtin `'dimensionless'` keyword that is equivalent to `""`. This was mean't to be a protection against no unit specified at all. I havent converted existing units yet